go/api/base_client.go

```diff
@@ -3492,3 +3495,24 @@ func (client *baseClient) XClaimJustIdWithOptions(
 	}
 	return handleStringArrayResponse(result)
 }
+
+// Returns the server time.
+//
+// Return value:
+// The current server time as a String array with two elements:
+// A UNIX TIME and the amount of microseconds already elapsed in the current second.
+// The returned array is in a [UNIX TIME, Microseconds already elapsed] format.
+//
+// For example:
+//
+//	result, err := client.Time()
+//	result: [{1737051660} {994688}]
+//
+// [valkey.io]: https://valkey.io/commands/time/
+func (client *baseClient) Time() ([]string, error) {
```

Comment:
Keep in mind different behavior on standalone and cluster clients - on standalone client calls `time` on the only primary node, in cluster - on a random primary.